src/Utilities/penalty_relaxation.jl

```diff
@@ -142,6 +142,7 @@ end
     PenaltyRelaxation(
         penalties = Dict{MOI.ConstraintIndex,Float64}();
         default::Union{Nothing,T} = 1.0,
+        no_warning_skip_constraint::Bool = false,
```

Comment:
```suggestion
        warn::Bool = true,
```
It'd be so much simpler if people opted out with `warn = false`.